src/commcare_cloud/python_migration_utils.py

```diff
@@ -0,0 +1,13 @@
+import six
+from io import open
+
+
+def open_for_json_dump(path):
+    """
+    This ensures compatible file write modes based on Python versions
+    Python 2 json.dump(s) returns a bytes object
+    Python 3 json.dump(s) returns a str object
```

Comment:
Did you mean `json.dumps()`
```suggestion

    Python 2 json.dumps(s) returns a bytes object
    Python 3 json.dumps(s) returns a str object
```